db/src/models/orders.rs

```diff
@@ -568,11 +568,11 @@ impl Order {
                 quantity: 1,
                 parent_id: None,
             };
-            if let Some(event_fee_in_cents) = event.fee_in_cents {
-                new_event_fee.unit_price_in_cents = event_fee_in_cents;
+            if event.fee_in_cents > 0 {
```

Comment:
You can remove this whole `if` statement and replace with 
```suggestion

 new_event_fee.unit_price_in_cents = event.fee_in_cents;
                new_event_fee.commit(conn)?;	                
```